src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -92,11 +138,16 @@ class pt5(Macro):
     
     def run(self, *args, **kwargs):
         pass
-
+ 
 class pt6(Macro):
     """Macro with a motor parameter followed by a list of numbers. The list as
     explicitly stated an optional last element which is a dictionary that defines the
-    min and max values for repetitions"""
+    min and max values for repetitions.
+    Usages, ex.:
+    pt6 mot1 [[1][34][1]]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.